configs/config.json.example

```diff
@@ -57,5 +57,28 @@
       "// any": {"keep_best_iv": 3},
       "// Example of keeping the 2 strongest (based on CP) and 3 best (based on IV) Zubat:": {},
       "// Zubat": {"keep_best_cp": 2, "keep_best_iv": 3}
-    }
+    },    
+    "vips" : {
+         "Any pokemon put here directly force to use Berry & Best Ball to capture, to secure the capture rate!": {},
+        "any": {"catch_above_cp": 1200, "catch_above_iv": 0.9, "logic": "or" },
```

Comment:
Should this be:
`"any": {"catch_above_cp": 0, "catch_above_iv": 0, "logic": "or"}, `